tests/Functional/Net/SCPSSH2UserStoryTest.php

```diff
@@ -18,7 +18,7 @@ class Functional_Net_SCPSSH2UserStoryTest extends PhpseclibFunctionalTestCase
     public static function setUpBeforeClass()
     {
         parent::setUpBeforeClass();
-        self::$remoteFile = uniqid('phpseclib-scp-ssh2-') . '.txt';
+        self::$remoteFile = uniqid('phpseclib-scp-ssh2-', true) . '.txt';
```

Comment:
I am pretty sure this change is irrelevant. I guess I have nothing against having it. I hate having to look at it in patches, though.